torch/_torch_docs.py

```diff
@@ -5000,13 +5000,14 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.linalg.matrix_power`
 """.format(**common_args))
 
-add_docstr(torch.matrix_exp,
-           r"""
+add_docstr(torch.matrix_exp, r"""
```

Comment:
Thank you for submitting this PR @neal2018! While we're fixing this doc let's take a couple opportunities to improve it, too.